src/components/greeting-container/GreetingContainer.tsx

```diff
@@ -17,11 +17,11 @@ const InfoContainer = styled.div`
   background: url(${logo}) no-repeat bottom right #d4edaa;
   box-shadow: 1.58105px 3.16211px 6.32421px rgba(0, 0, 0, 0.09);
```

Comment:
Not in this PR but why are we using float values for pixels? Seems a little funky

Reply:
I think it was just copy-pasted from the Figma design